index.html

```diff
@@ -1,41 +1,374 @@
+            </ul>
+            <p class="text">
+              A daily selection of privately personalized reads; no accounts
+              or sign-ups required.
+            </p>
+            <div class="card-exp">
+              <div class="Languages">
+                <div class="lan"><span>html</span></div>
+                <div class="lan"><span>css</span></div>
+                <div class="lan"><span>javaScript</span></div>
+              </div>
+            </div>
+            <div class="button">
+              <button class="btn" type="button">See Project</button>
+            </div>
+          </div>
+        </div>
+        <!--End of CARD 1-->
+        <!--Beginin of CARD 2-->
+        <div class="card card2">
+          <img class="card-photo" src="/Images/card2.png" alt="card2" />
+
+          <div class="card-content">
+            <h2 class="title">Multi-Post Stories</h2>
+            <ul class="list">
+              <li class="list-element">CANOPY</li>
+              <li class="dot"><span class="point"></span></li>
+              <li class="list-element text-grey">Back End Dev</li>
+              <li class="dot"><span class="point"></span></li>
+              <li class="list-element text-grey">2015</li>
+            </ul>
+            <p class="text">
+              A daily selection of privately personalized reads; no accounts
+              or sign-ups required.
+            </p>
+            <div class="card-exp">
+              <div class="Languages">
+                <div class="lan"><span>html</span></div>
+                <div class="lan"><span>css</span></div>
+                <div class="lan"><span>javaScript</span></div>
+              </div>
+            </div>
+            <div class="button">
+              <button class="btn" type="button">See Project</button>
+            </div>
+          </div>
+        </div>
+        <!--End of CARD 2-->
+        <!--Beginin of CARD 3-->
+        <div class="card card3">
+          <img class="card-photo" src="/Images/card3.png" alt="card3" />
+
+          <div class="card-content">
+            <h2 class="title">Tonic</h2>
+            <ul class="list">
+              <li class="list-element">CANOPY</li>
+              <li class="dot"><span class="point"></span></li>
+              <li class="list-element text-grey">Back End Dev</li>
+              <li class="dot"><span class="point"></span></li>
+              <li class="list-element text-grey">2015</li>
+            </ul>
+            <p class="text">
+              A daily selection of privately personalized reads; no accounts
+              or sign-ups required.
+            </p>
+            <div class="card-exp">
+              <div class="Languages">
+                <div class="lan"><span>html</span></div>
+                <div class="lan"><span>css</span></div>
+                <div class="lan"><span>javaScript</span></div>
+              </div>
+            </div>
```

Comment:
- [ ] Please implement the card technologies to be inside a `ul` tag instead of `div` tags, for semantic purposes. ( Please implement this to all cards )